src/platform/tests/TestConnectivityMgr.cpp

```diff
@@ -45,63 +45,37 @@ using namespace chip::DeviceLayer;
 //      Unit tests
 // =================================
 
-static void TestPlatformMgr_Init(nlTestSuite * inSuite, void * inContext)
+struct TestConnectivityMgr : public ::testing::Test
+{
+
+    static void SetUpTestSuite()
+    {
+        auto err = chip::Platform::MemoryInit();
+        EXPECT_EQ(err, CHIP_NO_ERROR);
+    }
+
+    static void TearDownTestSuite()
+    {
+        chip::Platform::MemoryShutdown();
+        chip::DeviceLayer::PlatformMgr().Shutdown();
```

Comment:
Yes, it is one of the tests right now. I thought about moving it to SetUpTestSuite, but I don't want to modify the logic of the current tests. I will add a comment that this is supposed to be moved to SetUpTestSuite. Or maybe do it now?

Reply:
This will have to be "fixed" once we port all tests and try to run them in randomized order. So it's a good idea to mark it as TODO in the code. However, I guess that we will find more cases like that anyway.